sdk/core/core-auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/core-auth",
-  "version": "1.8.1",
+  "version": "1.9.0",
```

Comment:
need to bump the dep on core-util to ensure it has `HttpMethods`

Reply:
done